### PR TITLE
Power up now spins on center axis, and all power up placements have b…

### PR DESCRIPTION
**Pull request by wbeltran777**
…een updated in residential and winterland

As the title says. Added a 'pivot' component in the tool box prefab so when the model spins it actually spins on it's center axis.

Also deleted all the level meshes since we don't need those anymore.

### CODE REVIEW

File: Assets/_PartyParkade/Script/Game/ItemPickUp.cs [hunk 1]
```diff
@@ -20,6 +20,8 @@ public class ItemPickUp : MonoBehaviour
         [SerializeField]
         private MeshRenderer m_pickupMesh = null;
         [SerializeField]
+        private Transform m_meshPivot = null;
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/Game/ItemPickUp.cs:23]: why did you need to add another variable?

Comment (m4rtint) [inline Assets/_PartyParkade/Script/Game/ItemPickUp.cs:23]: So a work around instead of updating the mesh? lol

Comment (mikeslgoh) [reply Assets/_PartyParkade/Script/Game/ItemPickUp.cs:23]: Did it not spin correctly with using just the mesh renderer?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/ItemPickUp.cs:23]: I'm guessing because of the mesh center point is off, we're making our own. Honestly thinking about, i'm half sure that you can probably get the "parent" which is probably the mesh pivot point?

Comment (wbeltran777) [reply Assets/_PartyParkade/Script/Game/ItemPickUp.cs:23]: Oh shoot I didn't realize there was another tool box mesh to replace it with

**Comment by mikeslgoh** [comment]
[Powerup placement (1.0)](https://trello.com/c/9BkG6QNn/710-powerup-placement-10)
